src/connect_to_peers.rs

```diff
@@ -289,6 +293,8 @@ where
         peer_distance,
     );
 
+    NUMBER_PEER_CONNECTIONS.fetch_add(1, std::sync::atomic::Ordering::SeqCst);
+
```

Comment:
We want to make it easy to verify that the number of entries in the `peer_map` and the value of `NUMBER_PEER_CONNECTIONS` is always in sync. To this end, let's add comments on top of all modifications to the `peer_map` where the corresponding modification to `NUMBER_PEER_CONNECTIONS` is happening. For this particular increment, please add the comment above the block 1449-1451 in `peer_loop.rs`/`PeerLoopHandler::run_wrapper`.